packages/compass-e2e-tests/helpers/commands/shell-eval.ts

```diff
@@ -57,6 +57,7 @@ export async function shellEval(
     } catch (err) {
       // just leave it unparsed for now if there's a parse error because
       // that's really helpful when debugging
+      console.error('Could not parse result:', result);
```

Comment:
Trying to figure out `AssertionError: expected 'TypeError: Cannot read property \'res…' to have property 'ok'`